packages/design-tokens/tokens/color.json

```diff
@@ -0,0 +1,64 @@
+{
+  "kz": {
+    "color": {
+      "wisteria": {
+        "100": "#F0F1F4",
+        "200": "#E1E2EA",
+        "300": "#C4C5D4",
+        "400": "#A7A9C1",
+        "500": "#898BA9",
+        "600": "#5B5E7E",
```

Comment:
Looking at the Zen UI Kit colours page in Figma, it has `#6B6E94` for Wisteria-600. Worth double checking maybe